src/components/MainNav/ProfileDropdown/ProfileDropdown.js

```diff
@@ -266,7 +266,7 @@ class ProfileDropdown extends Component {
     return (
       hasLabel ? (
         <>
-          <span className={css.button__label}>
+          <span className={css.button__label} aria-label={`${tenantName} ${servicePointName} profile`}>
```

Comment:
Hello @UladzislauKutarkin , @JohnC-80 Could you please clarify should we move this string to translate message for localization ` profile` and pass `tenantName` and `servicePointName` as value. Example https://github.com/folio-org/stripes-smart-components/blob/master/lib/ChangeDueDateDialog/LoanList.js#L99

Reply:
Agreed, @Dmitriy-Litvinenko: this message must be passed through react-intl just like the example you link to. This will allow "profile" to be correctly localized.

Reply:
Done! Good point

Reply:
@Dmitriy-Litvinenko please take a look again!